### PR TITLE
docs: merge Cube Core and Cube Cloud pages

### PR DESCRIPTION
**Pull request by hassankhan**

### CODE REVIEW

File: docs/content/Getting-Started/Migrate-from-Core/Import-GitHub-repository.mdx [hunk 1]
```diff
@@ -1,10 +1,10 @@
 ---
-title: 'Getting Started with Cube Cloud: Import GitHub repository'
+title: 'Import a GitHub repository to Cube Cloud'
```
Comment (igorlukanin) [inline docs/content/Getting-Started/Migrate-from-Core/Import-GitHub-repository.mdx:2]: @hassankhan Let's edit all "import" guides' titles this way. Reads better, right?

File: docs/content/Getting-Started/Migrate-from-Core/Import-GitHub-repository.mdx
Comment (igorlukanin) [inline docs/content/Getting-Started/Migrate-from-Core/Import-GitHub-repository.mdx:1]: @hassankhan Please take a look at the `getting-started-4.png` at the end of this file. It doesn't seem to exist.

File: docs/content/Configuration/Overview.mdx
Comment (igorlukanin) [inline docs/content/Configuration/Overview.mdx:1]: @hassankhan Let's move "Migrating from Express to Docker" and remove it from "Configuration" (I doubt it's relevant here)

File: docs/content/Configuration/Connecting-to-the-Database.mdx [hunk 2]
```diff
@@ -178,10 +184,28 @@ highlighted as such in their respective pages.
...
+deployment.
+
+### <--{"id" : "Cube Cloud"}--> Connecting to data sources not exposed over the Internet
+
+Cube Cloud supports connecting to data sources within private VPCs. Please
+[contact us][cube-contact] to learn more. If you already have VPCs
+enabled in your account, check out the [VPC documentation][ref-cloud-conf-vpc]
+to learn how to get started.
+
+[cube-contact]: https://cube.dev/contact/
```
Comment (igorlukanin) [inline docs/content/Configuration/Connecting-to-the-Database.mdx:209]: @hassankhan Given the length of the page, this is undiscoverable. Let's remove it from here and copy to every individual database page instead; some of them—e.g., Redshift—already have something along these lines.

File: docs/content/Configuration/Connecting-to-the-Database.mdx [hunk 3]
```diff
@@ -178,10 +184,28 @@ highlighted as such in their respective pages.
...
+address to your database. You can copy the IP address from either the Database
+Setup step in deployment creation, or from **Settings / Configuration** in your
+deployment.
+
+### <--{"id" : "Cube Cloud"}--> Connecting to data sources not exposed over the Internet
+
+Cube Cloud supports connecting to data sources within private VPCs. Please
+[contact us][cube-contact] to learn more. If you already have VPCs
+enabled in your account, check out the [VPC documentation][ref-cloud-conf-vpc]
+to learn how to get started.
```
Comment (igorlukanin) [inline docs/content/Configuration/Connecting-to-the-Database.mdx:207]: @hassankhan Given the length of the page, this is undiscoverable. Let's remove it from here and copy to every individual database page instead; some of them—e.g., Redshift—already have something along these lines.

File: docs/content/Reference/Frontend/@cubejs-client-ngx.mdx
Comment (igorlukanin) [inline docs/content/Reference/Frontend/@cubejs-client-ngx.mdx:1]: @hassankhan I see reference docs for Angular and Vue. Where's reference docs for React and `core` libraries?

Comment (hassankhan) [reply docs/content/Reference/Frontend/@cubejs-client-ngx.mdx:1]: Automatically generated via the `docs-gen` project

**Pull request merged**